### PR TITLE
Getting Ready for game build update

### PR DESCRIPTION
**Pull request by Irishstevie**
Getting Ready for 
Game Build 2802

## Checklist

- [ ] I have personally loaded this code into an updated qbcore project and checked all of its functionality.
- [ ] My code fits the style guidelines.
- [ ] My PR fits the contribution guidelines.
- [x] WIP (For game build 2802)

### CODE REVIEW

**Comment by Z3rio** [comment]
@Irishstevie DM me on discord when this is fully done / ready to be comitted and I will take a look at it! (Considering it says WIP in the PR).

File: shared/vehicles.lua [hunk 1]
```diff
@@ -5388,6 +5388,147 @@ QBShared.Vehicles = {
...
+        ['model'] = 'broadway',
+        ['price'] = 555000,
+        ['category'] = 'muscle',
+        ["categoryLabel"] = 'Muscle',
+        ['hash'] = `broadway`,
+        ['shop'] = 'UNRELEASED',
+    },
+    ['entity3'] = {                         --DLC +set sv_enforceGameBuild 2802
+        ['name'] = 'Entity MT',
+        ['brand'] = 'Overflod',
```
Comment (SirOMGitsYOU) [inline shared/vehicles.lua:5413]: Could you add the brand name as - Överflöd this just keeps it consistent I've already sent in a PR to fix this for the other that were also spelt like this :)

File: shared/vehicles.lua [hunk 2]
```diff
@@ -5388,6 +5388,147 @@ QBShared.Vehicles = {
...
+        ['brand'] = 'BF',
+        ['model'] = 'surfer3',
+        ['price'] = 354000,
+        ['category'] = 'vans',
+        ["categoryLabel"] = 'Vans',
+        ['hash'] = `surfer3`,
+        ['shop'] = 'UNRELEASED',
+    },
+    ['tahoma'] = {                         --DLC +set sv_enforceGameBuild 2802
+        ['name'] = 'Surfer Custom',
```
Comment (SirOMGitsYOU) [inline shared/vehicles.lua:5502]: This is the wrong car name, It should be - Tahoma Coupe

**Comment by SirOMGitsYOU** [comment]
```

    ['brickade2'] = {                         --DLC +set sv_enforceGameBuild 2802
        ['name'] = 'Brickade 6x6',
        ['brand'] = 'MTL',
        ['model'] = 'brickade2',
        ['price'] = 245000,
        ['category'] = 'service',
        ["categoryLabel"] = 'Service',
        ['hash'] = `brickade2`,
        ['shop'] = 'UNRELEASED',
    },
    ['everon2'] = {                         --DLC +set sv_enforceGameBuild 2802
        ['name'] = 'Hotring Everon',
        ['brand'] = 'Karin',
        ['model'] = 'everon2',
        ['price'] = 179000,
        ['category'] = 'sports',
        ["categoryLabel"] = 'Sports',
        ['hash'] = `everon2`,
        ['shop'] = 'UNRELEASED',
    },
```

Here are the only two missing from the list, I didn't bother adding the cargo plane for obvious reasons.

**Comment by SirOMGitsYOU** [comment]
New Pistol & Candy Cane for weapons.lua

```	
[`weapon_pistolxm3`] 			 = {['name'] = 'weapon_pistolxm3', 		['label'] = 'WM 29 Pistol', 			['ammotype'] = 'AMMO_PISTOL',	['damagereason'] = 'Pistoled / Blasted / Plugged / Bust a cap in'}, -- DLC 2802

[`weapon_candycane`] 			 = {['name'] = 'weapon_candycane', 		['label'] = 'Candy Cane', 			['ammotype'] = nil,	['damagereason'] = 'Melee killed / Whacked / Executed / Beat down / Murdered / Battered'}, -- DLC 2802
```

Items.lua for both 
```
['weapon_pistolxm3'] 			 = {['name'] = 'weapon_pistolxm3', 			 	['label'] = 'WM 29 Pistol', 			['weight'] = 5000, 		['type'] = 'weapon', 	['ammotype'] = 'AMMO_PISTOL',			['image'] = 'weapon_pistolxm3.png', 	['unique'] = true, 		['useable'] = false, 	['description'] = 'A small semi-automatic firearm designed to be held in one hand'}, -- DLC 2802

['weapon_candycane'] 			 = {['name'] = 'weapon_candycane', 			 	['label'] = 'Candy Cane', 				['weight'] = 5000, 		['type'] = 'weapon', 	['ammotype'] = nil,						['image'] = 'weapon_candycane.png', 	['unique'] = true, 		['useable'] = false, 	['description'] = 'A Candy Cane, Usually eaten.. But strangely looks good as a weapon'}, -- DLC 2802
```

I've also attached here 2 inventory images for both candy cane and pistol.

weapon_candycane
weapon_pistolxm3

**Comment by SirOMGitsYOU** [comment]
```
    ['virtue'] = {                         --DLC +set sv_enforceGameBuild 2802
        ['name'] = 'Virtue',
        ['brand'] = 'Ocelot',
        ['model'] = 'virtue',
        ['price'] = 1788000,
        ['category'] = 'super',
        ["categoryLabel"] = 'Super',
        ['hash'] = `virtue`,
        ['shop'] = 'UNRELEASED',
```

This should be Super car not sports :) - in your vehicles.lua its down as sports

**Review by Irishstevie — COMMENTED** [review]
I have been running on this build for a week or two everything seems ok but needs more testing :)

**Pull request closed without merging**